README.md

```diff
@@ -39,6 +39,21 @@ Contributors:
   * [Documentation Changes](/.github/PULL_REQUEST_TEMPLATE/docs.md)
   * [Test/Build/Other Changes](/.github/PULL_REQUEST_TEMPLATE/other.md)
 
+##### Compatibility Matrix
+
+| Controller-Runtime      | release-0.2 | release-0.3 | release-0.4 | release-0.5 |
+|-------------------------|-------------|-------------|-------------|-------------|
+| kubernetes/release-1.17 |     -       |     -       |     ✓       |     ✓       |
+| kubernetes/release-1.16 |     -       |     -       |     ✓       |     ✓       |
+| kubernetes/release-1.15 |     ✓       |     ✓       |     -       |     -       |
+| kubernetes/release-1.14 |     ✓       |     ✓       |     -       |     -       |
+| kubernetes/release-1.13 |     -       |     -       |     -       |     -       |
+| kubernetes/release-1.12 |     -       |     -       |     -       |     -       |
```

Comment:
I'd say we can probably remove these given that there is no support, wdyt?

Reply:
yeah by the policy, we provides support for three minor releases at a time. do you think if it's an improvement to clarify the compatibility for these out-of-support releases explicitly so that users wont even try spend time on it? given the fact that a portion of clusters aren't always catching up w/ the time windows of supports.. otherwise we can also rename it like `kubernetes/release-1.12 (unsupported)` for clarification?

Reply:
I don't think we necessarily need to for now, but also don't wanna block on it. When I first read the matrix I immediately noticed that the last two versions were there with no supported version. If we have  `1.14` as the first one, it'd probably going to be clear to users that's the minimum supported version

Reply:
convinced. i will shorten the rows to 1.14 on master. how do you think if i pick the matrix to release-0.[1-5]?:
- for matrix upon `release-0.1`, it will cover k8s `release-1.1[0-3]`
- for matrix upon `release-0.2`, it will cover k8s `release-1.1[1-4]`
- for matrix upon `release-0.3`, it will cover k8s `release-1.1[2-6]`
- for matrix upon `release-0.4`, it will cover k8s `release-1.1[3-6]`
- for matrix upon `release-0.5`, it will cover k8s `release-1.1[4-7]`